source/parser.hera

```diff
@@ -966,7 +966,10 @@ PipelineHeadItem
   OptimizedParenthesizedExpression
 
 PipelineTailItem
-  ( AwaitOp / Yield / Return / Throw ) !AccessStart !MaybeParenNestedExpression -> $1
+  ( AwaitOp / Return / Throw ) !AccessStart !MaybeParenNestedExpression -> $1
+  Yield $( _? Star )? !AccessStart !MaybeParenNestedExpression ->
```

Comment:
Could probably simplify to:

```
  $( Yield ( _? Star )? ) !AccessStart !MaybeParenNestedExpression ->
    return { $loc, token: $1, type: "Yield" }
```

This way will have slightly better source mapping.

Reply:
I thought `$loc` just had the start location, which is already in the return value of `Yield`? Or does it also have a length?

Reply:
Just checked and indeed `$loc` has a length. Fixed!